plugins/generator-1.20.1/forge-1.20.1/templates/projectile/projectile.java.ftl

```diff
@@ -120,6 +120,18 @@ public class ${name}Entity extends AbstractArrow implements ItemSupplier {
 	@Override public void tick() {
 		super.tick();
 
+        AABB bounding_box = new AABB(this.position(), this.position().add(this.getDeltaMovement()).inflate(${entity.modelWidth}f / 2, ${entity.modelHeight}f / 2, ${entity.modelWidth}f / 2));

```

Comment:
bounding_box is not really javanic name for variable

Reply:
ok I'll fix it